src/webapp/pages/settings/CreateButton.tsx

```diff
@@ -9,10 +9,11 @@ import {
     LandingPageEditDialog,
     LandingPageEditDialogProps,
 } from "../../components/landing-page-edit-dialog/LandingPageEditDialog";
-import { Description, OpenInBrowser } from "@material-ui/icons";
+import { Description, Message, OpenInBrowser } from "@material-ui/icons";
 
 export interface CreateButtonProps {
     landings: Maybe<LandingNode[]>;
+    newNotification: () => void;
```

Comment:
"newNotification" conveys a substantive, an object. If that's a callback that we will call when we have new notifications, maybe `onNewNotification`